src/database/adatabasesetup.cpp

```diff
@@ -302,12 +302,12 @@ bool ADataBaseSetup::backupOldData()
         return true;
     }
 
-    auto date_string = QDateTime::currentDateTime().toString(Qt::ISODate);
+    auto date_string = QDateTime::currentDateTime().toString("yyyy_MM_dd_T_hh_mm");
     auto backup_dir = QDir(AStandardPaths::absPathOf(AStandardPaths::DatabaseBackup));
-    auto backup_name = database_file.baseName() + "_bak_" + date_string + ".db";
+    auto backup_name = database_file.baseName() % "_bak_" % date_string + ".db";
```

Comment:
https://doc.qt.io/qt-5/qstring.html#more-efficient-string-construction
We can keep the fairly known `+` operator for concatenation but use qmake to our advantage for efficient string building

Reply:
i tried that, unfortunately using qstringbuilder is not compatible with the whole project at the moment